pyenvbuilder/main.py

```diff
@@ -0,0 +1,51 @@
+'''
+pyenvbuilder's commands
+'''
+import argparse
+import logging
+from pyenvbuilder import __version__
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def launch(log_level="INFO"):
+    '''
+    PyEnvBuilder's Launcher
+    '''
+    pyenvbuilder_logger = logging.getLogger('')
```

Comment:
Recently I learned that we should never attach and switch levels at the root logger ('').
Would you please change it to:
```
    pyenvbuilder_logger = logging.getLogger('pyenvbuilder')
```

This way when we change the level and also attach the handler we will do so just for our package and not for other dependent packages that we have (which would be affected if we use `.getLogger('')`.

Reply:
Ok great, thank you for letting me know, I implemented it